addons/mail/static/src/models/partner/partner.js

```diff
@@ -266,6 +274,9 @@ function factory(dependencies) {
         active: attr({
             default: true,
         }),
+        avatarUrl: attr({
+            compute: '_computeAvatarUrl',
+        }),
```

Comment:
would add `id` in `dependencies`, just to play safe and be consistent with the rest